### PR TITLE
Adding Tests for Command Line Pipeline

### PR DESCRIPTION
**Pull request by OwenMcGee**
Tests are currently not all implemented and those that are run too slowly

### CODE REVIEW

**Comment by mleotta** [comment]
jenkins test this please

**Comment by mleotta** [comment]
jenkins test this please

File: arrows/mvg/applets/CMakeLists.txt [hunk 1]
```diff
@@ -18,6 +18,11 @@ set( headers
 # Add our plugin path to the default list
 include_directories( ${CMAKE_CURRENT_BINARY_DIR} )
 
+if (KWIVER_ENABLE_TESTS)
+  add_subdirectory(tests)
+  list(APPEND kwiver_plugin_applets_subdir tests)
```
Comment (mleotta) [inline arrows/mvg/applets/CMakeLists.txt:23]: What does this line do?

Comment (OwenMcGee) [reply arrows/mvg/applets/CMakeLists.txt]: That line was a holdover from a previous attempt at writing the tests, I've now removed it.

**Review by mleotta — APPROVED** [review]
Approved, as long as CI passes

File: arrows/mvg/applets/tests/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+include(kwiver-test-setup)
+
+add_test(NAME mvg:track_features COMMAND kwiver track-features "${kwiver_test_data_directory}/aphill_240_1fps_crf32.mp4")
```
Comment (mleotta) [inline arrows/mvg/applets/tests/CMakeLists.txt:3]: We should probably change the test name to `mvg:applets:track_features`.

Let's not do this right now.  If the tests pass we can merge this and then make this change in the next PR when you add another test.

**Comment by mleotta** [comment]
jenkins test this please

**Pull request merged**